src/basic/efivars.c

```diff
@@ -241,7 +242,7 @@ int efi_set_variable(const char *variable, const void *value, size_t size) {
         if (futimens(fd, (struct timespec[2]) {
                                 { .tv_nsec = UTIME_NOW },
                                 { .tv_nsec = UTIME_NOW }
```

Comment:
Not a matter, but maybe it is better to define this as `TIMESPEC_NOW` with `static const`.

Reply:
Hmm, this only occurs once in the whole codebase, so I'm not sure?

But I'll make it `const`.